modules/csg/csg_shape.h

```diff
@@ -239,7 +239,7 @@ class CSGBox3D : public CSGPrimitive3D {
 	virtual CSGBrush *_build_brush() override;
 
 	Ref<Material> material;
-	Vector3 size = Vector3(2, 2, 2);
+	Vector3 size = Vector3(1, 1, 1);
```

Comment:
As long as it follows the style guide,
```suggestion
	Vector3 size(1, 1, 1);
```

Reply:
That's actually a style that we don't really use in the codebase so far, so it's better not to start using it as it adds inconsistency in the codebase.